src/ProjectTemplates/Web.ProjectTemplates/GrpcService-CSharp.Client.csproj.in

```diff
@@ -0,0 +1,21 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>netcoreapp3.0</TargetFramework>
+    <LangVersion>7.1</LangVersion>
```

Comment:
Is there any specific reason for not just specifying: `latest`?

Reply:
@jaredpar / @livarcocc - thoughts on what the templates ought to do?

Reply:
If this is for netcoreapp3.0 then I would delete the `LangVersion` element altogether. The compiler already recognizes this and moves the code to the latest language features (C# 8.0 preview). 

@cartermp

Reply:
Yep, you cam omit this. Starting with VS 2019 preview 4 it'll default to 8.0 preview

Reply:
My experience with 3.0 preview 2 is it doesn't. Has that changed in preview 3?

Reply:
The change isn't specific to .NET Core previews. It set to align with VS 2019 preview 4, and whichever corresponding .NET Core preview that has those bits will start to get the behavior.

Reply:
Let's keep this in this template for now, we can remove it in a subsequent preview if necessary. Nothing is broken by this right?

Reply:
Filed https://github.com/aspnet/AspNetCore/issues/7835.

Reply:
If this is a template for 3.0 P3, can't we just validate that 3.0 P3 has the behavior we want from roslyn and if that's the case clean up the template?

Reply:
Thinking out loud... I wonder how other tooling handles this? I've always had to set it to latest otherwise rider has complained in the past. Is there a doc someplace where tooling devs can look at and pick a default language version per specific target framework?

Reply:
@niemyjski No docs yet. We'll have a blog post out and update documentation fairly soon.

In terms of how other tooling handles this, it's entirely up to the tools on how to handle the default. For reference, this is done here in the .NET project system: https://github.com/dotnet/project-system/pull/4536

I imagine Rider will need to do similar work, though it likely isn't terribly complicated.

Reply:
Since we are in ask mode, If it's not broken I think we won't be making any changes. @livarcocc do you know of any scenarios that may be broken by this?